corehq/warehouse/models/dimensions.py

```diff
@@ -0,0 +1,86 @@
+from django.db import models
+
+
+class BaseDim(models.Model):
+    domain = models.CharField(max_length=255)
+
+    dim_last_modified = models.DateTimeField(auto_now=True)
+    dim_created_on = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        abstract = True
+
+
+class UserDim(BaseDim):
+    user_id = models.CharField(max_length=255)
+    username = models.CharField(max_length=255)
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+    email = models.CharField(max_length=255)
+    user_type = models.CharField(max_length=255)
```

Comment:
it's based on these: https://github.com/dimagi/commcare-hq/blob/master/corehq/pillows/utils.py#L23